common/syscalls/syscalls.h

```diff
@@ -656,6 +656,7 @@ REV_SYSCALL(  440, int rev_process_madvise(int pidfd, const struct iovec *vec, s
 
 REV_SYSCALL(  500, int rev_cpuinfo(struct rev_cpuinfo *info) );
 REV_SYSCALL(  501, int rev_perf_stats(struct rev_stats *stats) );
+REV_SYSCALL(  1024, int rev_open(const char *filename, int flags ) );
```

Comment:
Might want to keep it in numerical order, but with a blank line and a comment line explaining what it is for, so that it doesn't appear to just be another entry in the long list above it.